File: 412-FizzBuzz/412-FizzBuzz.js
// Last updated: 3/7/2026, 10:02:35 PM
/**
 * @param {number} n
 * @return {string[]}
 */
var fizzBuzz = function(n) {
    let answer = [];
    for(let i=0; i<n; i++){
        let num = i+1;
        answer[i] = `${num}`;
        if(num%3 == 0){
            answer[i] = "Fizz";
        }
        if(num%5 == 0){
            answer[i] = "Buzz";
        }
        if(num%3 == 0 && num%5 == 0){
            answer[i] = "FizzBuzz";
        }
    }
    return answer;
};

File: 412-FizzBuzz/412-FizzBuzz.cpp
// Last updated: 5/7/2026, 9:20:18 AM
1class Solution {
2public:
3    vector<string> fizzBuzz(int n) {
4        vector<string> res;
5
6        for(int i = 1; i <= n; i++) {
7
8            if(i % 3 == 0 && i % 5 == 0) {
9                res.push_back("FizzBuzz");
10            }
11            else if(i % 3 == 0) {
12                res.push_back("Fizz");
13            }
14            else if(i % 5 == 0) {
15                res.push_back("Buzz");
16            }
17            else {
18                res.push_back(to_string(i));
19            }
20        }
21
22        return res;
23    }
24};